Haskell/Tests/Simplicity/Programs/Tests.hs

```diff
@@ -2,7 +2,7 @@
 module Simplicity.Programs.Tests (tests) where
 
 import Prelude hiding (sqrt, all)
-import Control.Arrow ((***), right)
+import Control.Arrow ((***), (+++), right)
```

Comment:
fwiw I don't see the arrow choice combinator actually used anywhere.

Reply:
Seems it didn't make the cut in the end.

Fixed.